src/Amazon.AspNetCore.DataProtection.SSM/Amazon.AspNetCore.DataProtection.SSM.csproj

```diff
@@ -20,9 +20,18 @@
     <Company>Amazon Web Services</Company>
     <SignAssembly>true</SignAssembly>
     <SignOnBuild>true</SignOnBuild>
-    <Version>4.0.0-preview.1</Version>
-	<WarningsAsErrors>CA1727</WarningsAsErrors>
+    <Version>4.0.0-preview.2</Version>
```

Comment:
You shouldn't manually update the version moving forward. However, given that we are running the Workflows with overrides since we're in preview, then your change is harmless.